### PR TITLE
Add Helm chart

### PR DESCRIPTION
**Pull request by etiennetremel**

### CODE REVIEW

File: chart/estafette-gke-preemptible-killer/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: estafette-gke-preemptible-killer
+version: 1.0.35
+description: A Kubernetes controller that loop through a given preemptibles node pool and kill a node before the regular 24h life time of a preemptible VM
```
Comment (JorritSalverda) [inline chart/estafette-gke-preemptible-killer/Chart.yaml]: loop**s** through
kill**s** a node

File: chart/estafette-gke-preemptible-killer/values.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+
+image:
+  repository: estafette/estafette-gke-preemptible-killer
+  pullPolicy: IfNotPresent
+
+resources:
+  limits:
+    cpu: 50m
+    memory: 128Mi
+  requests:
```
Comment (JorritSalverda) [inline chart/estafette-gke-preemptible-killer/values.yaml]: Can you put request before limits? Slightly easier to read.

File: kubernetes.yaml [hunk 3]
```diff
@@ -46,6 +46,11 @@ spec:
         ports:
         - name: prom-metrics
           containerPort: 9001
+        env:
```
Comment (JorritSalverda) [inline kubernetes.yaml]: Is the time range between which the preemptibles get killed configurable as well?

File: README.md [hunk 4]
```diff
@@ -35,58 +35,37 @@ You can either use environment variables or flags to configure the following set
...
+export GO_PIPELINE_LABEL=1.0.35
+export DRAIN_TIMEOUT=300
+export INTERVAL=600
+export CPU_REQUEST=10m
+export MEMORY_REQUEST=16Mi
+export CPU_LIMIT=50m
+export MEMORY_LIMIT=128Mi
+
+# Setup RBAC
+curl https://raw.githubusercontent.com/estafette/estafette-gke-preemptible-killer/master/rbac.yaml -o rbac.yaml
```
Comment (JorritSalverda) [inline README.md]:
```
curl https://raw.githubusercontent.com/estafette/estafette-gke-preemptible-killer/master/rbac.yaml | envsubst | kubectl apply -n ${NAMESPACE} -f -
```

File: README.md [hunk 5]
```diff
@@ -35,58 +35,37 @@ You can either use environment variables or flags to configure the following set
...
+cat rbac.yaml | envsubst | kubectl apply -n ${NAMESPACE} -f -
+
+# Run application
+curl https://raw.githubusercontent.com/estafette/estafette-gke-preemptible-killer/master/kubernetes.yaml -o kubernetes.yaml
```
Comment (JorritSalverda) [inline README.md]:
```
curl https://raw.githubusercontent.com/estafette/estafette-gke-preemptible-killer/master/kubernetes.yaml | envsubst | kubectl apply -n ${NAMESPACE} -f -
```

File: kubernetes.yaml (hunk 3 above)
Comment (etiennetremel) [reply kubernetes.yaml]: It was not, I'm going to make it configurable later today

Comment (JorritSalverda) [reply kubernetes.yaml]: Okay, that can be done in a separate PR than. I'll merge this one.

**Pull request merged**